compiler/src/main/scala/dx/compiler/ExecutableTree.scala

```diff
@@ -5,6 +5,7 @@ import dx.core.ir.{Application, Callable, ExecutableKind, ExecutableLink, Workfl
 import dx.util.CodecUtils
 import spray.json._
 
+// APPS-622 links contains callee applets
```

Comment:
@jdidion   Actually, this is probably the callee generated applets in the tree of stuff being compiled -- will try some example w/ a native app dependency using dxni and look for that dependency, which should be annotated for APPS-622?